docs/web-apps/automated-testing/cypress.md

```diff
@@ -6,184 +6,148 @@ sidebar_label: Cypress
-    
-    It looks like this is your first time using Cypress: 4.9.0
-    
-    [18:49:28]  Verifying Cypress can run /home/seluser/.cache/Cypress/4.9.0/Cypress [started]
-    [18:49:33]  Verified Cypress!       /home/seluser/.cache/Cypress/4.9.0/Cypress [title changed]
-    [18:49:33]  Verified Cypress!       /home/seluser/.cache/Cypress/4.9.0/Cypress [completed]
-    
-    Opening Cypress...
-    
-    ================================================================================
-    
-      (Run Starting)
-   
-   ```
-  
-   
-  > You can also troubleshoot any issues with the [--verbose](/testrunner-toolkit/saucectl#verbose) flag.
-
-2. A Sauce Labs job link appears in the console where you can [verify the results in the Sauce Labs dashboard](#run-tests-on-sauce-labs):
-
-    ```bash
-    Preparing assets for tests/home.tree.test.js
-    
-    Open job details page: https://app.saucelabs.com/tests/8fb13276b39f40c2b05048776bcaeaf6
+    saucectl configure
     ```
-   
-## Run Tests on Sauce Labs
+1. Enter your Sauce Labs Username and Access Key at the prompts.
+1. Add `credentials.yml` to your `.gitignore` file to protect it from unintended exposure.
 
-The following steps outline how to run your cypress tests using Sauce Labs virtual machines:
-   
-1. Run your cypress tests with the following `saucectl` setting:
+### Set up Your Cypress Project
 
-    ```yaml
-    defaults:
-      mode: sauce
-    ```
-    _or_
+Using `saucectl` to run your Cypress tests and view them in the Sauce Labs app requires a directory structure that includes the Cypress project and the Sauce configuration keys. Follow the steps below to dynamically build the directory.
 
-    ```yaml
-    suites:
-      - name: saucy test
-        mode: sauce
+1. Run the `new` command:
     ```
-   
-   > For more information regarding the `saucectl` parameters, including how to increase your VM concurrency, please visit the [configuration](/testrunner-toolkit/configuration) documentation.
+    saucectl new
+    ```
+1. At the framework prompt, choose `Cypress`.
+1. At the region prompt, choose the Sauce Labs data center applicable to your account.
+
+At this point, `saucectl` generates a project that includes everything required to run a sample Cypress test through `saucectl`.
+
+If it detects a Cypress project already in the directory, `saucectl` will prompt you whether to overwrite any duplicates, such as the `cypress` folder and `cypress.json`. You can choose `No` to these questions to preserve your existing Cypress directory structure and files, and then manually [edit the `saucectl` configuration](/testrunner-toolkit/configuration) to run those tests.
+
+### Run Tests
+
+Use the `run` command to execute the sample test included with the `saucectl` installation or whichever test suites you have defined in the configuration file.
+
+```
+saucectl run
+```
+
+The console displays the executing tests, distinguishing which mode is running.
+
+```
+$ saucectl run
```

Comment:
Would remove console output-- this is constantly changing and makes doc too long